### PR TITLE
274 Tutorial for transferring tokens to a contract

### PR DESCRIPTION
**Pull request by ACStoneCL**
### Related links

[Tutorial for how to safely transfer token to a contract #274](https://github.com/casper-network/docs/issues/274)
[Documentation on how-to-use purses in a smart contract #286](https://github.com/casper-network/docs/issues/286)

### Changes

Added a "Safely Transferring Tokens to a Contract" tutorial, within the [Developers](https://casper.network/docs/dapp-dev-guide/) section under tutorials.

### Notes

Page ran locally without issue.

Tagged the tech writing team for proofreading/editting and Ed for review of content.

### CODE REVIEW

File: source/docs/casper/dapp-dev-guide/tutorials/transfer-token-to-contract.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Safely Transferring Tokens to a Contract
+
+This tutorial covers two methods through which you can handle tokens via a contract. There are a variety of potential applications that necessitate such a configuration. However, this is not a native process to the Casper network and will require the use of custom code. As such, the following two scenarios provide a framework for developers, as well as the pros and cons of each example. Developers should choose the option that best suits their individual needs.
+
```
Comment (Buddhika-Pathirathna) [inline source/docs/casper/dapp-dev-guide/tutorials/transfer-token-to-contract.md:4]: I feel like this section is a bit wordy and would suggest cutting down some of them and making sentences shorter.
like
This tutorial covers two methods to handle tokens via a contract. This is not a native process to the Casper network and will require the use of custom code. The following two scenarios provide a framework for developers and the pros and cons of each example. Developers should choose the option that best suits their individual needs.

**Review by ipopescu — COMMENTED** [review]
LGTM. I left minor suggestions for flow. I'll leave the final approval to @darthsiroftardis.

**Comment by ACStoneCL** [comment]
Thanks @Buddhika-Pathirathna and @ipopescu !

I've incorporated your feedback.

**Review by darthsiroftardis — APPROVED** [review]
LGTM
